regparser/commands/fill_with_rules.py

```diff
@@ -50,6 +52,21 @@ def process(tree_path, previous, version_id):
     (tree_path / version_id).write(new_tree)
 
 
+def merge_changes(document_number, changes):
```

Comment:
Are we merging patches _into_ changes or merging changes? Should it be called `merge_patches` instead?